libs/blocks/caas-config/caas-config.js

```diff
@@ -904,6 +946,17 @@ const Configurator = ({ rootEl }) => {
       });
   }, []);
 
+  useEffect(() => {
+    const setInitialState = async () => {
+      const initialState = await getInitialState();
+      dispatch({ type: 'SET_STATE', value: initialState });
+    };
+
+    if (!Object.keys(state).length) {
+      setInitialState();
+    }
+  }, [state]);
+
```

Comment:
Could you simplify this by putting an empty `[]` instead of `[state]` so the `useEffect` runs only once? Might be able to omit the check for state length on line 955 if so.